### ISSUE TITLE
Game freeze on loading video

### ISSUE THREAD

**Issue by MG4vQIs7Fv** [MaxLastBreath/TOTK-mods#34]
Thanks for the recent updates!
Things were running well for me until I triggered a video cutscene (Lightning Temple clear). I tried a few things to see if I could progress past this until I eventually unloaded all mods.
That worked and I figured I'd run into a bug with the Lightning Temple and moved on. Then a bit later the Blood Moon triggered and the game froze again.

The freeze occurs after the movie event triggers. Screen fades to black. A faint outline of the Switch loading icon appears but does not move.

When I tried to switch back to the Legacy upscale that seemed to fix the problem.

Edited (2023-10-28 17:53):
- TotK 1.1.2
- Yuzu EA 3910
- TotK Optimizer 1.4.3
- Arch Linux (6.5.5-273-tkg-pds)
- AppImage version of Yuzu and TotK Optimizer

```
[Settings]
font = Bahnschrift
color = cyan
shadow_color = light-blue
active_color = white
style = flatly
scale = On
backup = Off
cheat-backup = On
animation = On

[Updates]
dfps = New

[Options]
dfps version = New
resolution = 1440p QHD - Experimental
aspect ratio = Aspect Ratio 16-9
fps = No Change
shadowresolution = High x1024
ui = Playstation UI
first person = Off
camera speed x2 = On
remove quareduction = On
remove dof = Off
remove lensflare = On
improve lod = On
remove fxaa = Off
remove fsr = On

[Paths]

[Mode]
managermode = Yuzu
```

**Comment by MaxLastBreath** [comment]
Hi, it's caused by Max DFPS++ but it seems a bit inconsistent, I may be able to solve it but it has proven difficult to replicate for me. I'll look into it

**Comment by MG4vQIs7Fv** [comment]
Perhaps I can do some testing for you? I can reliably replicate the problem by viewing a "Memory" video from the game menu.

I forgot to add some details with the first post:
- Arch Linux (6.5.5-273-tkg-pds)
- AppImage version of both Yuzu and TotK Optimizer

**Comment by MaxLastBreath** [comment]
> Perhaps I can do some testing for you? I can reliably replicate the problem by viewing a "Memory" video from the game menu.
> 
> I forgot to add some details with the first post:
> 
> * Arch Linux (6.5.5-273-tkg-pds)
> * AppImage version of both Yuzu and TotK Optimizer

That would be really nice, I'll follow up with a potential fix soon :)

**Comment by MaxLastBreath** [comment]
Can you confirm if it happens on other versions as well or is it just 1.1.2, again i wasnt able to replicate it on 1.1.0 and 1.2.0 and 1.2.1 for now

**Comment by MaxLastBreath** [comment]
Found the issue, 1.1.2 had duplicate of the patch info, woops. I fixed it, expect the fix soon when i push the next major update for Max DFPS++, which is gonna be rather massive. So may take a bit of time.

**Comment by MaxLastBreath** [comment]
Has been officially fixed now, with 1.5.0 :) sorry took a while, as i was working on the UltraCam the entire time the past month
